plugins/in_process_exporter_metrics/pe.h

```diff
@@ -52,6 +52,7 @@ struct flb_pe {
     struct flb_input_instance *ins; /* input instance   */
     struct mk_list *metrics;        /* enabled metrics */
     int enabled_flag;               /* indicate enabled metrics */
+    int page_size;                  /* page size for converting number of pages to bytes */
```

Comment:
sysconf(3) returns long type of value. So, do we need to use long instead of int here?

Reply:
I fixed it